### PR TITLE
MAUI-3132 : Prepare migration document for sublayer in MAUI Maps control.

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
Hi @vigneshrameshsync 
I have prepared migration document for the sublayer in MAUI Maps control. please review it?

Regards,
Shalini

### CODE REVIEW

File: MAUI/Maps/migration.md [hunk 1]
```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
...
+   <td>
+      {{'[RangeColorMapping](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.RangeColorMapping.html)'| markdownify }}
+   </td>
+   <td>
+      {{'[RangeColorMapping](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.RangeColorMapping.html)'| markdownify }}
+   </td>
+</tr>
+</table>
+
+### Equal color mapping
```
Comment (vigneshrameshsync) [inline MAUI/Maps/migration.md:1956]: Don't use H3 heading, just bold it.

File: MAUI/Maps/migration.md [hunk 2]
```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
...
+      {{'[ShapeColorValuePath](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeColorValuePath)'| markdownify }} and {{'[ShapeValuePath](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeValuePath)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'ShapeColorValuePath'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the shape sublayer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
```
Comment (Chrispineagunja) [inline MAUI/Maps/migration.md:1468]: The following code example explains how to initialize the shape sublayer and customize the layer in the Xamarin SfMaps and .Net MAUI SfMaps.

File: MAUI/Maps/migration.md [hunk 3]
```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
...
+      {{'[ShapeStrokeThickness](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeStrokeThickness)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'StrokeThickness'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the layer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
```
Comment (Chrispineagunja) [inline MAUI/Maps/migration.md:2209]: The following code example explains how to initialize and customize the layer in the Xamarin SfMaps and .Net MAUI SfMaps.

File: MAUI/Maps/migration.md [hunk 4]
```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
...
+   <td>
+      {{'[-]'| markdownify }}
+   </td>
+   <td>
+      {{'StrokeDashArray'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the layer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
```
Comment (Chrispineagunja) [inline MAUI/Maps/migration.md:2400]: The following code example explains how to initialize and customize the layer in the Xamarin SfMaps and .Net MAUI SfMaps.

File: MAUI/Maps/migration.md [hunk 5]
```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
...
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'StrokeLineCap'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the layer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
```
Comment (Chrispineagunja) [inline MAUI/Maps/migration.md:2594]: The following code example explains how to initialize and customize the layer in the Xamarin SfMaps and .Net MAUI SfMaps.

File: MAUI/Maps/migration.md [hunk 6]
```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
...
+   <td>
+       {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'ControlPointFactor'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the layer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
```
Comment (Chrispineagunja) [inline MAUI/Maps/migration.md:2745]: The following code example explains how to initialize and customize the layer in the Xamarin SfMaps and .Net MAUI SfMaps.

File: MAUI/Maps/migration.md [hunk 7]
```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
...
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'Fill'| markdownify }}
+   </td>
+</tr>
+</table>
+
+The following code example explains how to intialize the layer and customize the layer in Xamarin SfMaps and .Net MAUI SfMaps.
```
Comment (Chrispineagunja) [inline MAUI/Maps/migration.md:2883]: The following code example explains how to initialize and customize the layer in the Xamarin SfMaps and .Net MAUI SfMaps.

File: MAUI/Maps/migration.md [hunk 8]
```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
...
+
+{% endhighlight %}
+
+{% endtabs %}
+</td></tr>
+</table>
+
 ## Upcoming features in .NET MAUI
 
 * Support for imagery layer such as Bing Maps, OpenStreetMap, etc.
```
Comment (Chrispineagunja) [inline MAUI/Maps/migration.md:2960]: * Support for imagery layers such as Bing Maps, OpenStreetMap, and more.

**Review by Chrispineagunja — APPROVED** [review]

File: MAUI/Maps/migration.md [hunk 9]
```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
...
+      {{'Points'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[ShapeFill](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html#Syncfusion_SfMaps_XForms_ShapeSetting_ShapeFill)'| markdownify }} in
+       {{'[ShapeSetting](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ShapeSetting.html)'| markdownify }} class
+   </td>
+   <td>
+      {{'[-](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapShapeLayer.html#Syncfusion_Maui_Maps_MapShapeLayer_ShapeFill)'| markdownify }}
```
Comment (vigneshrameshsync) [inline MAUI/Maps/migration.md:2369]: Remove this

File: MAUI/Maps/migration.md [hunk 10]
```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
...
+   <td>
+      {{'[Color](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ColorMapping.html#Syncfusion_SfMaps_XForms_ColorMapping_Color)'| markdownify }}
+   </td>
+   <td>
+      {{'[Color](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.ColorMapping.html#Syncfusion_Maui_Maps_ColorMapping_Color)'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[LegendLabel](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ColorMapping.html#Syncfusion_SfMaps_XForms_ColorMapping_LegendLabel)'| markdownify }}
```
Comment (vigneshrameshsync) [inline MAUI/Maps/migration.md:1932]: Remove this.

File: MAUI/Maps/migration.md [hunk 11]
```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
...
+<tr>
+<th>Xamarin SfMaps</th>
+<th>.NET MAUI SfMaps</th></tr>
+<tr>
+<td>
+{% tabs %}
+
+{% highlight xaml %}
+
+{% endhighlight %}
```
Comment (vigneshrameshsync) [inline MAUI/Maps/migration.md:2606]: Remove empty tabs and add hyphen (-)

File: MAUI/Maps/migration.md (hunk 1 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: Changed

File: MAUI/Maps/migration.md [hunk 12]
```diff
@@ -1363,11 +1363,1602 @@ The following code example explains how to customize tooltip in the Xamarin SfMa
...
+   <td>
+      {{'[Value](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.EqualColorMapping.html#Syncfusion_SfMaps_XForms_EqualColorMapping_Value)'| markdownify }}
+   </td>
+   <td>
+      {{'[Value](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.EqualColorMapping.html#Syncfusion_Maui_Maps_EqualColorMapping_Value)'| markdownify }}
+   </td>
+</tr>
+</table>
+
+### Range color mapping
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: Changed

File: MAUI/Maps/migration.md (hunk 2 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: Modified

File: MAUI/Maps/migration.md (hunk 3 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: Modified

File: MAUI/Maps/migration.md (hunk 4 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: Modified

File: MAUI/Maps/migration.md (hunk 5 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: Modified

File: MAUI/Maps/migration.md (hunk 6 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: Modified

File: MAUI/Maps/migration.md (hunk 7 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: Modified

File: MAUI/Maps/migration.md (hunk 8 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md:2920]: Modified

File: MAUI/Maps/migration.md (hunk 9 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: Removed

File: MAUI/Maps/migration.md (hunk 10 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: removed

File: MAUI/Maps/migration.md (hunk 11 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md]: Removed

**Review by vigneshrameshsync — APPROVED** [review]
The content is fine.

**Pull request merged**